demos/python_demos/common/models/__init__.py

```diff
@@ -15,8 +15,16 @@
 """
 
 
-from .ssd import SSD
-from .yolo import YOLO
-from .faceboxes import FaceBoxes
 from .centernet import CenterNet
+from .faceboxes import FaceBoxes
 from .retinaface import RetinaFace
+from .ssd import SSD
+from .yolo import YOLO
+
+__all__ = [
+    'CenterNet',
+    'FaceBoxes',
+    'RetinaFace',
+    'SSD',
+    'YOLO',
+]
```

Comment:
This is not needed:
> [It also includes any submodules of the package that were explicitly loaded by previous import statements.](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package)

Reply:
I'm aware, but the problem is that flake8 will consider the imports unused if the imported names aren't... well... _used_.